Sources/ProjectDescription/Dependencies/CocoapodsDependencies.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/// Contains the description of dependencies that can be installed using CocoaPods.
+public struct CocoaPodsDependencies: Codable, Equatable {
```

Comment:
@danyf90 / @laxmorek what do you think about this interface? Do you foresee the need for other attributes here?

Reply:
Hey @pepicrft I think we can kick off this feature as MVP where list of pods may be enough. We will be extending it in the future.